nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -312,7 +339,7 @@ def __init__(
                 # No preprocessing, the output is the audio signal
                 audio_len = dali.fn.shapes(dali.fn.reshape(audio, shape=[-1]))
                 audio = dali.fn.pad(audio)
-                self.pipe.set_outputs(audio, audio_len, transcript, transcript_len)
+                self.pipe.set_outputs(audio, audio_len, indices)
             else:
                 # Additive gaussian noise (dither)
                 if self.dither > 0.0:
```

Comment:
I think you can replace `normal_distribution` with `random.normal` to get rid of the deprecation warning.

Reply:
Isnt `audio` the result of a a dali pipeline step? Would python's random.normal work with the pipeline output like this? For now, ill keep it as is and patch it in a later PR.

Reply:
I was talking about L346 - `dali.fn.normal_distribution` is replaced by `fn.random.normal` now (we moved it to a dedicated namespace).